packages/vite/src/node/server/index.ts

```diff
@@ -285,6 +285,7 @@ export async function createServer(
   const moduleGraph = new ModuleGraph(container)
   const closeHttpServer = createServerCloseFn(httpServer)
 
+  // eslint-disable-next-line prefer-const
```

Comment:
Why is this needed?

Reply:
Down in line https://github.com/vitejs/vite/pull/2733/files#diff-abb3345b6e3b2ec6d297c2ebc54cca85ae4487a31bac3cc9e78457f5114adb26R353 it gets assigned
I don't know if it's a bug of eslint that it doesn't detect it correctly

Reply:
Looks like a bug, indeed. This seems a valid use of let according to the rule.